tests/it/c/compile/runtime_args.d

```diff
@@ -56,6 +56,31 @@ import it;
     }
 }
 
+@("--ignore-macros")
+@safe unittest {
```

Comment:
Does this fail without the change? And why would that be?

Reply:
Yes, it fails because the preamble has some `#define` statements that are not preprocessed when `--ignore-macros` is used. 

```
pallocator.d(13,16): Warning: C preprocessor directive `#define` is not supported
pallocator.d(13,9): Error: declaration expected, not `#`
pallocator.d(15,12): Warning: C preprocessor directive `#define` is not supported
```

pallocator.d preamble:
```
   1 module pallocator;
   2
   3
   4         import core.stdc.config;
   5         import core.stdc.stdarg: va_list;
   6         static import core.simd;
   7         static import std.conv;
   8
   9         struct Int128 { long lower; long upper; }
  10         struct UInt128 { ulong lower; ulong upper; }
  11
  12         struct __locale_data { int dummy; }  // FIXME
  13         #define __gnuc_va_list va_list
  14
  15     #define __is_empty(_Type) dpp.isEmpty!(_Type)
```